main.py

```diff
@@ -61,6 +64,17 @@ def main():
         if fullscreen:
             tdl.set_fullscreen(not tdl.get_fullscreen())
 
+        # Enemy turn
+        for entity in display_manager.cur_map.entities:
+            entity.take_turn(player, display_manager.cur_map)
```

Comment:
we could eventually do something like if target not in fov -> move randomly

Reply:
Sure, good idea.